classes/blocks/class-gallery.php

```diff
@@ -53,7 +53,7 @@ public function register_gallery_block() {
 				'render_callback' => static function ( $attributes ) {
 					$attributes['images'] = self::get_images( $attributes );
 
-					return self::render_frontend( $attributes );
+					return self::hydrate_frontend( $attributes );
```

Comment:
We need to create a similar `render_frontend` method because when you `hydrate` a component you need to pass the `data-hydrate` instead of a `data-render` attribute. After we `hydrate` all `render` components we'll need to remove [this method](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/932/files#diff-b0e752e847c025f266c46aadf983955b5796c943773645980d725d628c73120fR233) too.

Reply:
Would this affect how we implemented hydration for the Carousel Header? Maybe not in this PR, but we could open a follow-up ticket to investigate that

Reply:
We would need to check that, good suggestion :)